gluten-core/src/main/scala/io/glutenproject/execution/FileSourceScanExecTransformer.scala

```diff
@@ -180,7 +180,8 @@ class FileSourceScanExecTransformer(@transient relation: HadoopFsRelation,
   }
 
   override def doValidate(): Boolean = {
-    if (BackendsApiManager.getTransformerApiInstance.supportsReadFileFormat(relation.fileFormat)) {
+    if (BackendsApiManager.getTransformerApiInstance.supportsReadFileFormat(relation.fileFormat) &&
+    !bucketedScan) {
```

Comment:
Does it imply scanning bucketing table is not supported by all backends? It's better to add some notes to mark this as an unsupported feature.

Reply:
Bucketing table has `bucketId` in filename https://github.com/apache/spark/blob/master/sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/BucketingUtils.scala#L34, I suppose CH does not treat this case. Added notes.